cms/templates/index.html

```diff
@@ -34,7 +34,7 @@ <h3 class="sr">${_("Page Actions")}</h3>
           <a href="mailto:${settings.FEATURES.get('STUDIO_REQUEST_EMAIL','')}">${_("Email staff to create course")}</a>
           % endif
 
-          % if show_new_library_button:
+          % if library_creator_status=='granted':
```

Comment:
For courses, the various status values (granted, pending, unrequested, denied) exist to manage the workflow for users to request creator privileges. For libraries, that workflow is not separated though, so I think the code is nicer with a simple boolean flag here (as it was before), than checking equality against some string constant (`'granted'`). In other words: If a variable has only two values, and one is a yes value and one is a no value, I think you should just use a boolean.